src/plugins/data/public/query/query_string/language_service/types.ts

```diff
@@ -66,4 +67,7 @@ export interface LanguageConfig {
   supportedAppNames?: string[];
   hideDatePicker?: boolean;
   sampleQueries?: SampleQuery[];
+  addFiltersToQuery?: (query: string, filters: Filter[]) => string;
+  /** Add filters to the natural language prompt. */
+  addFiltersToPrompt?: (prompt: string, filters: Filter[]) => string;
```

Comment:
this seems like it if we passed a query object with a language we could do some switch case instead of defining the same function twice

Reply:
@joshuali925

Reply:
Last time i checked the Query object doesn't contain prompt, prompt is not a query